### PR TITLE
deb and rpm support

### PR DESCRIPTION
**Pull request by vandot**
Packages will be created when we release new version, until then it is possible to test build locally.
Install latest `goreleaser` (submitted couple of PRs to support our build) and run `goreleaser release --skip-publish --rm-dist`

### CODE REVIEW

File: packaging/bee.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Bee configuration - https://swarm-gateways.net/bzz:/docs.swarm.eth/docs/installation/configuration
+
+api-addr: :1633
```
Comment (janos) [inline packaging/bee.yaml:3]: This should be commented as default, when ports pr is merged.

File: packaging/bee.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# bootnode:
+# - /dnsaddr/bootnode.ethswarm.org
+# clef-signer-enable: false
+# clef-signer-endpoint: ""
+config: /etc/bee/bee.yaml
+# cors-allowed-origins: []
+data-dir: /var/lib/bee
+# db-capacity: "5000000"
+debug-api-addr: :1635
```
Comment (janos) [inline packaging/bee.yaml:12]: This should be commented as default, when ports pr is merged.

File: packaging/bee.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+# debug-api-enable: false
+# gateway-mode: false
+# global-pinning-enable: false
+# help: false
+# nat-addr: ""
+# network-id: "1"
+p2p-addr: :1634
```
Comment (janos) [inline packaging/bee.yaml:19]: This should be commented as default, when ports pr is merged.

**Review by janos — APPROVED** [review]
Very nice. Thanks!

**Review by acud — APPROVED** [review]

**Pull request merged**